client/service-worker-manager.js

```diff
@@ -18,30 +18,60 @@ function urlBase64UrlToUint8Array(base64UrlData) {
   return buffer;
 }
 
+/**
+ *  get service worker registration
+ */
+function getServiceworkerRegistration() {
```

Comment:
Do we really need this function? I didn't found where this is used.
Moreover, what is the advantage of this function rather than getRegistration()?

```
getServiceWorkerRegistraiton().then(registration => {
  // do something
});
```
```
navigator.serviceWorker.getRegistrations().then(registration => {
  // do something
});
```